platform/linux-dpdk/pool/dpdk.c

```diff
@@ -91,12 +90,12 @@ int odp_pool_init_global(void)
 	return 0;
 }
 
-int odp_pool_init_local(void)
+static int dpdk_pool_init_local(void)
```

Comment:
I think it is better to keep the empty functions, especially for the data plane functions, so that we do not have to have NULL checks at call sites.